client/src/templates/Challenges/components/fill-in-the-blanks.tsx

```diff
@@ -32,12 +32,21 @@ function FillInTheBlanks({
     return '';
   };
 
+  const getInputLabel = (index: number) => {
+    if (answersCorrect[index] === true)
+      return t('learn.fill-in-the-blank.correct-answer');
+    if (answersCorrect[index] === false)
+      return t('learn.fill-in-the-blank.incorrect-answer');
+
+    return t('learn.fill-in-the-blank.blank');
+  };
```

Comment:
I don't think you'll need this if you take my advice to leave the accessible name on these as "blank" at all times.